src/test/java/com/intuit/graphql/filter/common/EmployeeDataFetcher.java

```diff
@@ -96,4 +113,9 @@ public String getSqlExpression() {
     public Specification<Object> getSpecification() {
         return specification;
     }
+
+    public Criteria getCriteria() {
```

Comment:
Its better to rename this method as `getMongoCriteria()`

Reply:
Its done.